src/test/resources/pages/instructorFeedbackResultsAddComment.html

```diff
@@ -358,7 +358,7 @@
                               <li class="list-group-item list-group-item-warning" id="responseCommentRow-0-1-0-1-1">
                                 <div id="commentBar-0-1-0-1-1">
                                   <span class="text-muted">
-                                    From: Teammates Test [${datetime.now}]
+                                    From: Teammates Test [Fri, 30 Jun 2017, 05:24 AM +0800]
```

Comment:
You need to update the `replaceUnpredictableValuesWithPlaceholders` method in HtmlHelper. As the time zone of the comment time has changed, the HtmlHelper is unable to replace occurrences of current time correctly. One solution I can think of now is to loop through all possible time zones and replace all possible current time representations with the placeholder. Would be good if you can find a better solution :joy: